src/EPR.Calculator.Frontend/Models/DefaultSchemeParameters.cs

```diff
@@ -7,23 +7,23 @@ public class DefaultSchemeParameters
     {
         public int Id { get; set; }
 
-        public string ParameterYear { get; set; }
+        public required string ParameterYear { get; set; }
 
         public DateTime EffectiveFrom { get; set; }
 
         public DateTime? EffectiveTo { get; set; }
 
-        public string CreatedBy { get; set; }
+        public string? CreatedBy { get; set; }
```

Comment:
CreatedBy will always contain a value.

Reply:
CreateBy can not be set as required without updating MockData. 

![image](https://github.com/user-attachments/assets/8f98ee97-4a2b-4f5e-9fdf-9b5c5b7a833b)

What should be set in mock values?

Reply:
Updated the mock data.